restapi/src/main/java/io/stargate/web/resources/v2/RowsResource.java

```diff
@@ -147,10 +147,7 @@ public Response get(
             where = buildWhereForPath(tableMetadata, path);
           } catch (IllegalArgumentException iae) {
             return Response.status(Response.Status.BAD_REQUEST)
-                .entity(
-                    new Error(
-                        "not enough partition keys provided",
-                        Response.Status.BAD_REQUEST.getStatusCode()))
+                .entity(new Error(iae.getMessage(), Response.Status.BAD_REQUEST.getStatusCode()))
```

Comment:
@dougwettlaufer  I think that exception msg was just swallowed, correct?

Reply:
In this situation I don't think the raw error needs to be returned to the user since we're returning a 400 specifically because of the partition keys.

Reply:
Same for below changes.